modules/spock/src/main/groovy/org/testcontainers/spock/SpockTestDescription.groovy

```diff
@@ -0,0 +1,36 @@
+package org.testcontainers.spock
+
+import groovy.transform.PackageScope
+import org.spockframework.runtime.extension.IMethodInvocation
+import org.testcontainers.lifecycle.TestDescription
+
+/**
+ * Spock specific implementation of a Testcontainers TestDescription.
+ *
+ * Filesystem friendly name is based on Specification and Feature.
+ */
+@PackageScope
+class SpockTestDescription implements TestDescription {
+
+    String specName
+    String featureName
+
+    static SpockTestDescription fromTestDescription(IMethodInvocation invocation) {
+        return new SpockTestDescription([
+            specName: invocation.spec.name,
+            featureName: invocation.feature.name
+        ])
+    }
+
+    @Override
+    String getTestId() {
+        return null
```

Comment:
huh?

Reply:
This method is used nowhere, TBH I don't even understand why we have it in our interface and assumed it to come from refactoring from original JUnit test description.

Reply:
we could/should deprecate it, but let's not return `null` here anyways.

Reply:
What is semantically correct here then? Same then filesystemFriendly name?

Reply:
should work, yes 👍